notifications_utils/polygons.py

```diff
@@ -164,13 +164,16 @@ def simplify(self):
 
     @cached_property
     def bleed(self):
```

Comment:
What do you see as the use of this `bleed` property going ahead? Will we use it in the admin app ever (or should we?) or will it become redundant and instead we will use `bleed_by` always?

I'm cautious of having two similarly named methods, one that gives you a default bleed and one that gives you a custom bleed. I would prefer to have a single method that has say a default I think...

Reply:
Good call, I’ve removed the property in ea6393c